Sources/ProjectDescription/TargetDependency.swift

```diff
@@ -2,11 +2,16 @@ import Foundation
 
 // MARK: - TargetDependency
 
+public enum SDKStatus: String {
```

Comment:
Would you mind adding documentation to this one? Keep in mind that the models in the `ProjectDescription` framework are available from the manifest file so developers might want to use the documentation to know more about the models that they are using.